### PR TITLE
Translate text

### PR DESCRIPTION
**Pull request by hmangukia**
Say "translate _text to be translate_ to _language to be translated in_" 
For example, "translate how are you to chinese" or "translate hallo welt to english"
Note: English, Spanish, French, German, Italian and Portuguese are supported

### CODE REVIEW

File: windows/activities.md [hunk 1]
```diff
@@ -129,6 +130,7 @@ NOTE: All the recorded files will be saved on the desktop. Make sure that the fi
 * "***get tweets***" Get latest 10 tweets from timeline. For this an app is to be registered at https://apps.twitter.com/ (if confused, have a look at https://spring.io/guides/gs/register-twitter-app/). A new file named "twitterCredentials.py" is to be made in the windows folder. Save the consumer key, consumer secret, access token, access token secret with the names consumer_key, consumer_secret, access_token, access_secret repectively in the "twitterCredentials.py". Speak/type "get tweets" and top 10 tweets will be shown, also when and by whom it was created.
 * Get latest 10 tweets done by the user. Speak/type "get my tweets" and top 10 tweets done by the user(you) will be shown with the time when it was tweeted
 * You can get a list of everyone you follow from twitter by saying "get list of people I follow on twitter"
+* "***translate text***" Say "translate" and then the text you want to translate. For example, say "translate how are you to french" or "translate hallo welt to english". (Note: English, Spanish, French, German, Italian and Portuguese are supported)
```
Comment (sara-02) [inline windows/activities.md]: `you to French`  .... `to English`
Both should be capitalised

File: windows/benji.py [hunk 2]
```diff
@@ -73,17 +75,38 @@ def events(frame,put):
...
+		elif link[-1] == "french":
+			id = "fr"
+		elif link[-1] == "german":
+			id = "de"
+		elif link[-1] == "italian":
+			id = "it"
+		elif link[-1] == "portugese" or link[-1] == "portuguese":
+			id = "pt"
+		translated = translator.translate(pystring, src=lang, dest=id)
+		print(translated.text)
```
Comment (sara-02) [inline windows/benji.py]: Can we speak out the translated text as well? Printing is also fine

**Comment by hmangukia** [comment]
Have made the changes @sara-02 :)

**Comment by hmangukia** [comment]
Done @sara-02

**Comment by the-ethan-hunt** [comment]
@hmangukia please fix your conflicts. Otherwise the PR is good to merge

**Comment by hmangukia** [comment]
Resolved the conflicts @the-ethan-hunt

**Pull request merged**